registration/tests/views/test_family.py

```diff
@@ -79,6 +81,84 @@ def test_post_family(self):
         )
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
 
+    def test_search_familes_no_params(self):
+        url = reverse("family-list") + "search/"
+        self.client.force_authenticate(self.user)
+        response = self.client.get(url)
+
+        self.family.parent = self.parent
+        self.family.save()
```

Comment:
Are these lines are necessary for the test? Because the request returns 400 from just having no query parameters.